modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -22,10 +22,7 @@ import ClayEmptyState from '@clayui/empty-state';
 import FrontendDataSetContext from './FrontendDataSetContext';
 import ManagementBar from './management_bar/ManagementBar';
 import CreationMenu from './management_bar/components/CreationMenu';
-import {
-	getFilterSelectedItemsLabel,
-	getOdataFilterString,
-} from './management_bar/components/filters/Filter';
+import {FILTER_IMPL_MAP} from './management_bar/components/filters/Filter';
```

Comment:
`Impl` smells a bit like backend :smile: 

But, I do like the refactor. A map is more explicit, compared to misleadingly simple functions.